go-packages/meep-mq/mq.go

```diff
@@ -0,0 +1,273 @@
+/*
```

Comment:
This package implements meep-mq as a singleton where all message queues are maintained (e.g. local + global)
Expectations were that this is implemented as a mq-store like functionality, were multiple mq objects can be created.

Reply:
Updated meep-mq package to create new instances for each message queue based on queue name, and owner module name & namespace.